mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -104,8 +107,8 @@ private[classification] trait LogisticRegressionParams extends ProbabilisticClas
    * @group setParam
    */
   def setThresholds(value: Array[Double]): this.type = {
-    if (isSet(threshold)) clear(threshold)
-    set(thresholds, value)
+    logWarning("Ignoring setThresholds(), use setThreshold() for binary Logistic Regression.")
```

Comment:
I updated the top line of the note to reiterate that this functionality will be re-enabled, and maintained the second line as thresholds can still be set via a param map with {model name}.thresholds -> {Array(values)}. Do you think it makes more sense to remove the top line instead?